python/img_hdf5_reader.py

```diff
@@ -107,6 +108,58 @@ def to_pngs(self, channel_clips, sample, json_file):
         # This JSON file is not used right now:
         # really just a list of the files processed.
 
+    def to_ometiff(self, channel, sample, ometif_path, clip, ometif_basename):
```

Comment:
Why the unused parameters? Why `to_ometiff`? It seems to be returning a numpy array.